src/components/Facets.tsx

```diff
@@ -0,0 +1,88 @@
+import { Facets as FacetsCompoundComponent } from './Filters/Facets';
+import { FilterGroup } from './FilterGroup';
+import { CompositionMethod } from '../hooks/useComposedCssClasses';
+import { Fragment } from 'react';
+
+/**
+ * The CSS class interface for {@link Facets}.
+ *
+ * @public
+ */
+export interface FacetsCssClasses {
+  container?: string,
+  divider?: string,
+  searchInput?: string,
+  optionsContainer?: string,
+  option?: string,
+  optionInput?: string,
+  optionLabel?: string
+}
+
+/**
+ * Props for the {@link Facets} component.
+ *
+ * @public
+ */
+export interface FacetsProps {
+  /** Whether or not the filter is collapsible. Defaults to true. */
```

Comment:
can we inherit most of these docs from the new FilterGroup component to avoid duplication (and similarly for StaticFiltersProps)?

Reply:
sure thing